dev/STRINGExplore_PHALY-Example.R

```diff
@@ -0,0 +1,31 @@
+#### Versions ####
```

Comment:
Two options: move this to `./inst/scripts` after merge, or cover it in the vignette. `./dev/` is only for actual development.